### DISCUSSION TITLE
Utenti mac - accesso remoto di HA da browser

### DISCUSSION THREAD

**Discussion by Macretti** [indomus/forum#2316] [Networking]
Bg a tutti,

ho HA core (docker) su raspberry PI 3 B+ (raspbian) + caddy e sottodominio duckdns. 

Da smartphone e altri dispositivi l'accesso remoto funziona e sembra ben configurato. Su mac (che conosco veramente poco...) accedo in remoto (in loopback) solo da Firefox (che riconosce il certificato), gli altri browser (chrome, che uso, ma anche opera, safari) mostrano sempre l'errore:

"Il sito non può fornire una connessione protetta
www.*****.duckdns.org ha inviato una risposta non valida.
ERR_SSL_PROTOCOL_ERROR"

senza alcuna (apparente) possibilità di navigazione, cosa mi sfugge?

**Comment by snakuzzo** [comment]
Perché stai sbagliando il protocollo.
Probabilmente quando digiti la URL stai usando HTTP al posto di HTTPS

**Reply by Macretti** [reply]
grande mia svista. galeotto fu il [www](http://www/)... alcuni browser lo mettono di default. io mi concentravo a guardare il protocollo e il www passava inosservato. ovviamente https://www.pippo.duckdns.org:1234/ è diverso da https://pippo.duckdns.org:1234/, basta solo farci caso. cmq grazie per la dritta, non sbagliavo il protocollo, ma il dominio.
